staff/victori-murillo/pre-course/02_calculator-pro/index.js

```diff
@@ -0,0 +1,46 @@
+var numbers = []
+
+function calculator() {
+  var num = ""
+  var moreNumbers = false
+
+  do {
+    num = prompt("Ingrese un numero")
+
+    if (!isNaN(num) && num.length > 0) {
+      numbers.push(Number(num))
+      moreNumbers = confirm("Quieres ingresar otro numero?")
+
+    } else if (num.length === 0) {
+      alert("Ningun dato ingresado")
+      num = NaN
```

Comment:
It probably might not be necessary to declare num as `NaN` as being undefined or null or a string will make the function `isNaN` return `false`

Reply:
I'm gonna see, if not break the code i'll do it